### ISSUE TITLE
Have rules about resolving discussions / conversations in code review

### ISSUE THREAD

**Issue by arogachev** [yiisoft/docs#153]
PR examples:

- https://github.com/yiisoft/form/pull/184

Notes for PR's author:

- When Github suggestions functionality is used - if it's fixed exactly as suggested, resolving it is OK (it happens automatically). Otherwise it still needs to be rechecked by reviewer. 
- Closing without doing anything (either leaving a comment or making a fix) is not good.

Why it's so important? It's needed to save time for opening collapse blocks and analyzing the changes. And in case of a bigger amount of discussions, it's even longer.

Maybe discussions should be only resolved by it's author?

Another important things to keep in mind:

- Do a research if it can be automated, at least with help of external plugins, etc.
- This should be user friendly for non-core contributors.
- Add this to internal docs somewhere.

**Comment by arogachev** [comment]
Another idea is to mark what is fixed completely and partially with different text tags / emojis to be able to quickly search and navigate through discussions using browser native tools.

**Comment by arogachev** [comment]
Emojis are images though and search only works in HTML.

**Comment by arogachev** [comment]
This may sound like a good idea, but I'm closing this because of the following reasons:

- It doesn't fit open source specifics and our team in particular.
- If PR already got sufficient amount of approves, according discussions automatically become non-blocking.
- Increases complexity, especially for non core team members.
- If it's possible for discussion to be closed - it will be closed. Github has no functionality to disallow it for now,
- Discussed with @samdark and the conclusion was that it won't work (based on experience).

I'll take it into consideration and switch my style of handling discussions too.

Bottom line:

if you think discussion is resolved - resolve it yourself.
